components/automate-cli/pkg/verifyserver/server/api/v1/startMockServer.go

```diff
@@ -0,0 +1,70 @@
+package v1
+
+import (
+	"errors"
+	"fmt"
+	"net/http"
+
+	"github.com/chef/automate/components/automate-cli/pkg/verifyserver/models"
+	"github.com/gofiber/fiber"
+)
+
+func (h *Handler) StartMockServer(c *fiber.Ctx) {
+
+	reqBody := new(models.StartMockServerRequestBody)
+	err := c.BodyParser(&reqBody)
+
+	// If request body is invalid
+	if err != nil {
+		errString := fmt.Sprintf("start mock-server request body parsing failed: %v", err.Error())
+		h.Logger.Error(fmt.Errorf(errString))
+		c.Next(&fiber.Error{Code: http.StatusBadRequest, Message: "Invalid request body"})
+		return
+	}
+
+	// If port number is invalid
+	err = validatePortRange(reqBody.Port, h)
+	// fmt.Println("My Error", err.Error())
+	if err != nil {
+		h.Logger.Error(err)
+		c.Next(&fiber.Error{Code: http.StatusBadRequest, Message: "Invalid port number"})
+		return
+	}
+
+	// Server is already running in the port
+	if err = validatePortAlreadyInUse(reqBody.Port, h); err != nil {
+		h.Logger.Error(err)
+		c.Next(&fiber.Error{Code: http.StatusConflict, Message: fmt.Sprintf(`"%s" server is already running on port %d`, reqBody.Protocol, reqBody.Port)})
+		return
+	}
+
+	err = h.MockServersService.StartMockServer(reqBody)
+
+	if err != nil {
+		errString := fmt.Sprintf("start mock-server request body contains unsupported protocol: %v", err.Error())
+		h.Logger.Error(fmt.Errorf(errString))
+		c.Next(&fiber.Error{Code: http.StatusBadRequest, Message: "UnSupported Protocol"})
```

Comment:
Check this error once because in case server does not start for any reason then also it returns UnSupported Protocol which is not correct message

Reply:
Updated